components/eam/src/chemistry/utils/tracer_data.F90

```diff
@@ -1602,6 +1638,114 @@ subroutine read_za_trc( fid, vid, loc_arr, strt, cnt, file, order )
     end if
 !    if(dycore_is('LR')) call polar_average(loc_arr)
   end subroutine read_za_trc
+!------------------------------------------------------------------------
+  subroutine read_za_trc_linoz( fid, vid, loc_arr, strt, cnt, file, order ,vid_srf)
```

Comment:
What does "za" stand for?

Reply:
za stands for zonal average, which fits our inputdata. Currently, only our linoz data use this function.